doc/modules/mod_jingle_sip.md

```diff
@@ -42,3 +104,168 @@ MongooseIM 2.2.x packages are built with OTP 19.3, so they include Jingle/SIP su
 * `listen_port` (default: 5600) the port on which MongooseIM will listen for incomming SIP messages
 * `local_host` (default: "localhost") value used to create SIP URIs (including VIA headers)
 * `sdp_origin` (default: "127.0.0.1") value of the `c=` SDP attribute
+
+The simplest configuration is the following:
+
+```erlang
+{mod_jingle_sip, []}
+```
+
+With this configuration MongooseIM will try sending SIP messages to a SIP proxy listening on localhost and port 5060.
+
+### Use cases covered by tests
+
+Currently to test the functionality we use a SIP Proxy mock written in Erlang.
+The following scenarios are covered by our tests in `big_tests/tests/jingle_SUITE.erl`
+
+
+All the sequence diagrams where generated with [textart.io/sequence](https://textart.io/sequence).
+The source code is embedded in the markdown file below every diagram inside a comment `<!--- --->`
+
+
+#### 1. Establishing a session with other XMPP user
+
+With the mod_jingle_sip enabled, all Jingle IQ set stanzas listed above are intercepted, translated to SIP packets and sent to a SIP Proxy.
+This means that current implementation will also translate stanzas addressed to a user in the same domain.
```

Comment:
+This means that the current implementation will also translate stanzas addressed to a user in the same domain.